stable/datadog/templates/NOTES.txt

```diff
@@ -28,5 +28,47 @@ Then run:
 {{- end }}
 
 {{- if .Values.datadog.apmEnabled }}
-The datadog agent is listening on ports 7777 and 8126.
+The datadog agent is listening on port 8126.
+{{- end }}
+
+{{- if (and (.Values.daemonset.enabled) (.Values.deployment.enabled)) }}
+
+#################################################################
+####               WARNING: Deprecation notice               ####
+#################################################################
+
+Simultaneous DaemonSet + Deployment installation within a single
+release will be deprecated in a future version. You will still be
+able to run both as separate releases with separate value files.
+
+Running as a Deployment is not necessary for event collection
+anymore, thanks to the leader election feature.
+
+We encourage you to migrate to a single DaemonSet, or split your
+chart if your use case still requires a separate Deployment.
+{{- end }}
+
+{{- if eq .Values.image.repository "datadog/docker-dd-agent" }}
+
+#################################################################
+####               WARNING: Deprecation notice               ####
+#################################################################
+
+The autoconf value is deprecated, Autodiscovery templates can now
+be safely moved to the confd value. As a temporary measure, both
+values were merged into the {{ template "datadog.confd.fullname" . }} configmap,
+but this will be removed in a future chart release.
+Please note that duplicate file names may have conflicted during
+the merge. In that case, the confd entry will take precedence.
+{{- end }}
+
+{{- if eq .Values.image.repository "datadog/docker-dd-agent" }}
```

Comment:
If I use `--set image.repository=datadog/docker-dd-agent` I will see both the deprecation notice and the error message. It seems confusing to have both of these?

Reply:
Good catch, the agent5 error should trigger at the toplevel, same as the missing apikey. reworking that